### PR TITLE
refactor(nixos.wiki): replace with wiki.nixos.org

### PR DESCRIPTION
**Pull request by somerand0mcat**
## 🔧 What does this fix? 🔧
as said in the title
image

## 🗒 Checklist 🗒

- [X] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [X] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

File: styles/nixos.wiki/README.md
Comment (uncenter) [inline styles/nixos.wiki/README.md:1]: Don't generate the auto-generated files please.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: styles/nixos.wiki/README.md
Comment (somerand0mcat) [reply styles/nixos.wiki/README.md:1]: i didnt generate the file

Comment (uncenter) [reply styles/nixos.wiki/README.md:1]: Look at the first line of that README file.

Comment (somerand0mcat) [reply styles/nixos.wiki/README.md:1]: oh

Comment (somerand0mcat) [reply styles/nixos.wiki/README.md:1]: i guess i'll just leave the readme file alone

Comment (uncenter) [reply styles/nixos.wiki/README.md:1]: Update the userstyles.yml file, and rename this userstyle's directory.

File: scripts/userstyles.yml [hunk 1]
```diff
@@ -638,7 +638,7 @@ userstyles:
     readme:
       app-link: "https://search.nixos.org"
     current-maintainers: [*cecelot]
-  nixos.wiki:
+  nixos-wiki:
```
Comment (uncenter) [inline scripts/userstyles.yml:641]: Also update the URL below.

**Comment by somerand0mcat** [comment]
btw, im not going to respond to anything in the PR untill tomorrow because i dont have much time so feel free to add the "waiting for author" label

**Comment by isabelroses** [comment]
Some issuess on the revisions page: https://wiki.nixos.org/w/index.php?title=NixOS_Wiki&action=history
image

Some readablity and unthemed sections here: https://wiki.nixos.org/wiki/Special:WhatLinksHere/NixOS_Wiki
image

Can we import a syntax highlighting from somewhere? see code example: https://wiki.nixos.org/wiki/Cheatsheet

The login page
image

**Comment by somerand0mcat** [comment]
isn't syntax highlighting already themed?

**Comment by isabelroses** [comment]
> isn't syntax highlighting already themed?

not as far as i can see, viewing https://wiki.nixos.org/wiki/Cheatsheet

**Review by isabelroses — APPROVED** [review]

**Comment by somerand0mcat** [comment]
im going to rewrite this port again because the previous code from 2 weeks ago doesn't look good to me

**Comment by somerand0mcat** [comment]
i was going to update/rewrite some parts of this userstyle because there were a bunch of unthemed stuff but i decided not to because it's really hard to code, theres alot of things that are hard to theme (e.g search bar, account preferences, etc)

**Pull request closed without merging**